### ISSUE TITLE
[FEATURE] ReturnYouTubeDislike

### ISSUE THREAD

**Issue by TheMaestro1s** [libre-tube/LibreTube#38]
**ReturnYouTubeDislike API integration**
Bring back the dislike button.

**Additional context**
no

**Comment by rimthekid** [comment]
How should we do that

**Comment by BrightDV** [comment]
Check [NewPipe Sponsor Block.](https://github.com/polymorphicshade/NewPipe)
They have recently added integration for it in [this pull request.](https://github.com/polymorphicshade/NewPipe/pull/127)

**Comment by alefvanoon** [comment]
@FireMasterK Do you have any plans to add it to piped-backend?

**Comment by FireMasterK** [comment]
No, the rate limit is too low and RYD's backend is closed source still.

**Comment by brownshark1993** [comment]
You've probably already seen this, but there's a fork of NewPipe with SponsorBlock and ReturnYouTubeDislike - https://github.com/polymorphicshade/NewPipe. Might be helpful
